bin/lib/localized-urls.json

```diff
@@ -137,7 +135,6 @@
         "cards.petLink": "/pdfs/cards/zh-tw/PetCards.pdf",
         "cards.pongLink": "/pdfs/cards/zh-tw/PongCards.pdf",
         "cards.raceLink": "/pdfs/cards/zh-tw/RaceGameCards.pdf",
-        "cards.starterLink": "/pdfs/cards/zh-tw/Scratch2Cards.pdf",
```

Comment:
Whoops, looks like both copies were removed for zh-tw. We still want this one I think.